### DISCUSSION TITLE
oii, tudo bem? sou iniciante em java e to com dúvida sobre funções.

### DISCUSSION THREAD

**Discussion by melsantiago** [frontendbr/forum#2188] [Perguntas e Respostas]
é meu primeiro post aqui, então perdão se to fazendo algo errado. as questões são essas e preciso respondê-las em js.

1. Faça uma função que recebe por parâmetro um valor inteiro e positivo e retorna o valor
lógico Verdadeiro (true) caso o valor seja primo e Falso (false) em caso contrário. 

2. Faça uma função que recebe a idade de um nadador por parâmetro e retorna, a
categoria desse nadador de acordo com a tabela abaixo:
Idade Categoria
5 a 7 anos: Infantil A
8 a 10 anos: Infantil B
11 a 13 anos: Juvenil A
14 a 17 anos: Juvenil B
Maiores de 18 anos (inclusive): Adulto

3. Faça uma função que recebe a média final de um aluno por parâmetro e retorna o seu
conceito, conforme a tabela abaixo:
Nota Conceito
De 0,0 a 4,9: D
De 5,0 a 6,9: C
De 7,0 a 8,9: B
De 9,0 a 10,0: A

4. Construa um algoritmo que contenha a seguinte estrutura:

a. Contenha uma função denominada AreaCirculo. Essa função recebe como
parâmetro o raio de um círculo e retorna a área desse círculo.

b. Contenha uma função denominada CompCirculo. Essa função recebe como
parâmetro o raio de um círculo e retorna o comprimento desse círculo.

c. Escreva o código de um programa que leia 5 valores que representam os raios
de cinco círculos. Para cada valor informado o programa deve imprimir a área e
o comprimento (perímetro) do círculo. Para obter o valor da área do círculo o
programa deve chamar a função AreaCirculo. Para obter o valor do seu
perímetro o programa deverá chamar a função CompCirculo.
Fórmulas:
Área de um círculo 𝐴=𝜋𝑟²
Perímetro de um círculo 𝑃=2𝜋�

**Comment by NayRSilva** [comment]
Olá Mel, como você é iniciante, uma coisa que é interessante quando você busca ajuda em fórum é escrever suas dúvidas explicitamente e não apenas colar as questões, assim as pessoas podem te ajudar com suas dúvidas pontuais. Também é boa prática colar o código com o qual você está tendo problemas para compartilhar seu raciocínio com os outros e assim receber ajuda mais facilmente.

Agora,  se você está precisando de explicações mais gerais ou tutoriais, recomendo fazer primeiro uma pesquisa por tutoriais mesmo pelo google. Se você precisa fazer esses exercícios em javascript, uma boa documentação com explicações e exemplos é a MDN Web Docs do Mozilla, que tem conteúdo em português.
